ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -17,6 +17,9 @@ public extension Colors {
         public struct Filled {
             public static var title = UIColor(light: textOnAccent, dark: Outline.title)
         }
+        public struct Gradient {
```

Comment:
this name is misleading. the button itself isn't gradient

Reply:
reverted the change and exposed the background color overriding in pillbuttonbar.

---
In reply to: [465861096](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r465861096) [](ancestors = 465861096)

Reply:
updated with new approach in latest iteration

---
In reply to: [465893014](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r465893014) [](ancestors = 465893014)